### PR TITLE
Configuring first article and author block

### PR DESCRIPTION
**Pull request by ardalis**

### CODE REVIEW

**Comment by madskristensen** [comment]
Hi Steve,

Completely unrelated to this PR, I just wanted you to see http://vwd.azurewebsites.net/ where we are building a feature showcase for the web features in VS (domain and hosting TBD). Later on we might want to talk about cross linking etc.

**Comment by ardalis** [comment]
Thanks, I'll keep an eye on it.

File: docs/yourfirst/yourfirstaspnetapplication.rst [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+.. _`Steve Smith`: Author_
+
+ASP.NET 5 provides a host of improvements over its predecessors, including improved performance, better support for modern web development standards and tools, and improved integration between WebAPI, MVC, and WebForms.  In addition, you can easily develop ASP.NET 5 applications using a variety of tools and editors, but Visual Studio continues to provide a very productive way to build web applications.  In this article, we'll walk through creating your first ASP.NET 5 web application using Visual Studio 2015.
+
+This article covers the following topics:
+	- Create a new ASP.NET 5 Project
+	- The wwwroot folder
+	- Resolving Client-Side Dependencies
+	- Resolving Servier-Side Dependencies
```
Comment (Eilon) [inline docs/yourfirst/yourfirstaspnetapplication.rst]: Servier == French Server? :smile:

File: docs/yourfirst/yourfirstaspnetapplication.rst [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+Looking at the files in the root of the project, we may notice the absence of a few files. Global.asax is no longer present, nor is web.config, both mainstays from the start of ASP.NET. Instead, we find a Startup.cs file and a config.json file.  Adding to this mix are bower.json, gruntfile.js, package.json, and project.json (the Project_Readme.html file you can see in the browser tab). Clearly the success of Javascript in web development has had an effect on how ASP.NET 5 projects are configured and compiled, and deployed, with JavaScript Object Notation (JSON) files replacing XML for configuration purposes.
+
+While we're at it, you may not notice it from the Solution Explorer, but if you open Windows Explorer you'll see that there is no longer a .csproj file, either. Instead you'll find a .kproj file **(ED: which is still XML and my understanding is that it may be going away before RTM – update this section prior to RTM)**.
```
Comment (Eilon) [inline docs/yourfirst/yourfirstaspnetapplication.rst]: No plans to get rid of krpoj or that it's MSBuild as far as I know. We're renaming the extension from kproj to xproj but that's it to my knowledge.

Comment (ardalis) [inline docs/yourfirst/yourfirstaspnetapplication.rst]: Ok, great, thanks!

**Pull request merged**